include/control_toolbox/pid.hpp

```diff
@@ -134,16 +131,14 @@ class Pid
    * \param i_min The min integral windup.
    * \param antiwindup If true, antiwindup is enabled and i_max/i_min are enforced
    *
-   * \throws An std::invalid_argument exception is thrown if i_min > i_max
```

Comment:
Same here

May be we should use other than zero for the sake for invalid initialization and then throw if the user sets a bad value?

Reply:
same here :D 
We throw only if the PID class is created with wrong values, not if the gains are set later.

Reply:
Right now we are not throwing anything right? I mean if it is set wrongly on constructor?

Reply:
Anyway, this is something not related to this PR. May be we can come back and check later

Reply:
I haven't changed the behavior here, just updated the docstrings.

You are here watching at the `Pid::Gains` struct, and we don't throw because it is also used in `Pid::set_gains` which should not throw an exception.
But we throw at construction of the PID class (see `Pid::Pid` constructor)

Reply:
Gotcha!